app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    files = command[3:].split(" ")
+    source_name = files[0]
+    dest_path = files[1]
+    if "/" in dest_path:
```

Comment:
Avoid using a hardcoded "/" as a directory separator in the script. instead, use a platform-independent method to ensure the destination includes folders.